lesson 5/OpenCVtest/Source.cpp

```diff
@@ -0,0 +1,51 @@
+#include "tensor.h"
+#include <iostream>
+#include "Exception.h"
+#include <opencv2/core/core.hpp>
+#include <opencv2/highgui/highgui.hpp>
+#include <opencv2/imgproc/imgproc.hpp>
+#include <cassert>
+
+using namespace cv;
+
+
+void assert_equal_mat(const cv::Mat& a, const cv::Mat& b) {
```

Comment:
I fixed this function, look at my changes and pull them 
https://github.com/nikolaiEGS/CPPDaser/pull/5/commits/d20e6b6f50edc1055790603b1d9a275ea6f0f827